scripts/import/import_EPMC.pl

```diff
@@ -277,7 +295,7 @@ sub get_epmc_data{
     return undef unless defined $id && $id =~/\d+/;
 
     my $xs   = XML::Simple->new();
-    my $server = 'http://www.ebi.ac.uk/europepmc/webservices/rest/';
+    my $server = 'https://www.ebi.ac.uk/europepmc/webservices/rest/';
```

Comment:
Would it make sense to move this "$server" variable outside the method in order to avoid having to create this variable each time we call the "get_epmc_data" method ?